index.html

```diff
@@ -455,24 +455,26 @@
       <div class="modal-content">
         <div class="well">
           <h3>Deinterlace a video</h3>
-          <p><code>ffmpeg -i <i>input_file</i> -c:v libx264 -vf "yadif,format=pix_fmts=yuv420p" <i>output_file</i></code></p>
+          <p><code>ffmpeg -i <i>input_file</i> -c:v libx264 -vf "yadif,format=yuv420p" <i>output_file</i></code></p>
           <p>This command takes an interlaced input file and outputs a deinterlaced H.264 MP4.</p>
           <dl>
             <dt>ffmpeg</dt><dd>starts the command</dd>
             <dt>-i <i>input file</i></dt><dd>path, name and extension of the input file</dd>
             <dt>-c:v libx264</dt><dd>tells ffmpeg to encode the video stream as H.264</dd>
             <dt>-vf</dt><dd>video filtering will be used<dd>
             <dt><i>"</i></dt><dd>start of filtergraph (see below)</dd>
-            <dt><i>yadif</i></dt><dd>deinterlacing filter (‘yet another deinterlacing filter’)</dd>
+            <dt><i>yadif</i></dt><dd>deinterlacing filter (‘yet another deinterlacing filter’)<br>
+              By default, <a href="https://ffmpeg.org/ffmpeg-filters.html#yadif-1">yadif</a> will output one frame for each frame. Outputting one frame for each <em>field</em> (thereby doubling the frame rate) with <code>yadif=1</code> may produce visually better results.</dd>
```

Comment:
I guess we used the typographic tags `<i>` and `<b>` instead of the semantic ones `<em>` etc., but I’m fine either.